extensions/adobe/experience/intelligentServices/profile-journeyai-sendtimeoptimization.schema.json

```diff
@@ -44,7 +44,7 @@
                       "meta:descriptionId": "profile-journeyai-sendtimeoptimization##https://ns.adobe.com/experience/intelligentServices/sendTimeScoreForClick##description##72351"
                     },
                     "https://ns.adobe.com/experience/intelligentServices/sendTimeScoreForOpen": {
-                      "title": "Send Time Score for Open",
+                      "title": "Send Time Score for Open (to be deprecated)",
```

Comment:
Please add "meta:status": "deprecated" to the property to deprecate the property.